wftests/warrior_tests/suites/rest_functional_tests/rest_json_test_Suite.xml

```diff
@@ -40,5 +40,26 @@
 			<onError action="next"/>
 			<impact>impact</impact>
 		</Testcase>
+		<Testcase>
+			<path>../../testcases/rest_functional_tests/tc_env_variable_substitution_in_verify_response.xml</path>
+			<context>positive</context>
+			<runtype>sequential_keywords</runtype>
+			<onError action="next"/>
+			<impact>impact</impact>
+		</Testcase>
+		<Testcase>
+			<path>../../testcases/rest_functional_tests/tc_not_preseneted_env_variable_substitution_in_verify_response.xml</path>
+			<context>positive</context>
+			<runtype>sequential_keywords</runtype>
+			<onError action="next"/>
+			<impact>noimpact</impact>
+		</Testcase>
+		<Testcase>
+			<path>../../testcases/rest_functional_tests/tc_not_preseneted_env_variable_substitution_in_verify_response.xml</path>
+			<context>positive</context>
+			<runtype>sequential_keywords</runtype>
+			<onError action="next"/>
+			<impact>noimpact</impact>
```

Comment:
This shouldn't be noimpact instead the context for failing testcase-steps should be flipped.